### DISCUSSION TITLE
Does BRAW Toolbox work on an AMD Threadripper Hackintosh?

### DISCUSSION THREAD

**Discussion by nickdriftwood** [latenitefilms/BRAWToolbox#138] [Support]
Just wondered if this would work on AMD threadripper hackintosh. We have to do a work around for intel dylibs. Thanks.

**Comment by latenitefilms** [comment]
It's a Universal App, so it SHOULD work fine on a Hachintosh. It definitely works fine on older Cheese Grater Mac Pro's. If Resolve works, then in theory, BRAW Toolbox should work too.

**Comment by nickdriftwood** [comment]
Thanks Chris
Great news that it should be supported.

I'm wanting to write about this app for my Lumix for S Series Group on Facebook with 10k users. I am Chief admin.

Is there ny chance I could trial it and promote it for you?

I already mentioned it in February this year as a ‘important release’. https://www.facebook.com/groups/332322654258440/permalink/1374916756665686/

Nick

**Reply by latenitefilms** [reply]
There's no real facility on the Mac App Store to trial things unfortunately. If you give me your Mac App Store email, I can try add you to a TestFlight build though.

**Comment by nickdriftwood** [comment]
Thanks Chris
Lets give it a whirl. Its ***@***.*** ***@***.***>


Best

Nick

**Reply by latenitefilms** [reply]
It seems GitHub removes your email for privacy. Shoot me through an email: support@latenitefilms.com

**Reply by latenitefilms** [reply]
Got it - added. It's an older v1.1.3 (50) build.
